vyper/builtin_functions/functions.py

```diff
@@ -120,6 +121,22 @@
 SHA256_BASE_GAS = 60
 SHA256_PER_WORD_GAS = 12
 
+class FoldedFunction(BuiltinFunction):
+    # Base class for nodes which should always be folded
+    _id = ""
```

Comment:
probably can leave this out -- that way if the subclass does not supply `_id`, accessing the attribute will fail explicitly rather than silently

Reply:
Good point!